dashboard/app/views/lti_mailer/lti_integration_confirmation.html.haml

```diff
@@ -1,7 +1,7 @@
 %p
-  = t('lti.integration.mailer_confirmation_body').html_safe
+  != t('lti.integration.mailer_confirmation_body', integration_guide_url: CDO.studio_url('/'), course_offering_url: CDO.studio_url('/'), markdown: true)
 %br
 %p
-  = t('lti.integration.mailer_confirmation_signature').html_safe
+  = t('lti.integration.mailer_confirmation_signature')
```

Comment:
@Hamms Would this string need to use the markdown option too?

Reply:
Based on [this mailer template/view](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/teacher_mailer/hour_of_code_tutorial_email.html.haml), it seems for strings we're not injecting variables into didn't need the markdown option, but I'm happy to add it in.

Reply:
Actually, looking at this again, it looks like I'm missing the `I18n` prefixing the `.t`. The view renders correctly, I'm not actually sure if it's needed or not?

Reply:
In views, we can just use the t:
https://github.com/code-dot-org/code-dot-org/blob/3734b38a8e833726d832df1fc5803392fe0764cc/i18n/locales/source/dashboard/base.yml#L9

Reply:
Quick note, just for edification: injecting variables with string interpolation and the markdown option are two independent pieces of functionality that you can apply to strings. We need to use the markdown option when the underlying string contains markdown syntax and needs to be formatted as such, regardless of whether or not we also use interpolation variables.